### PR TITLE
Node.js Foundation Momentum Release

### PR DESCRIPTION
**Pull request by ZibbyKeaton**
I was also hoping to post this momentum release to the website. It came out during Node.js Interactive, so it's a little late... But, I want it on the site for historically reference. Please let me know if you have any questions, and again sorry for my errors in advance.

### CODE REVIEW

**Comment by bnb** [comment]
@ZibbyKeaton When you say it came out, do you mean it was published to the Medium account? I feel like I remember seeing it, but I can't find it anywhere. 🤔

**Comment by ZibbyKeaton** [comment]
@bnb I should have clarified, we issued this over Marketwired. We have yet to post it on our website. I did, however, post a Node.js Interactive blog on our Medium channel that highlights pieces of this news release. Does that make sense?

**Comment by mikeal** [comment]
@bnb it was also sent to journalists directly upon release.

**Comment by hackygolucky** [comment]
oh hey friends! Have the answers Zibby provided been deemed sufficient enough for us to merge this as an archived press release? Gave this a few days for response.

**Comment by lpinca** [comment]
@hackygolucky there are a couple of nits in the markdown syntax that should be fixed before this can be merged.

**Comment by lpinca** [comment]
Am I allowed to push on Zibby's branch to fix them?

**Comment by hackygolucky** [comment]
@lpinca that would be delightful. Please do.

**Comment by lpinca** [comment]
Done, this is what I changed:

- Removed commit message body as it was not relevant to the change.
- Renamed file to "nodejs-foundation-momentum-release.md" to be consistent with the others.
- Fixed post metadata at the beginning of the file.
- Added a sentence at the end ("The Node.js Foundation is a Linux Foundation Project, which are independently funded software projects that harness the power of collaborative development to fuel innovation across industries and ecosystems. [www.linuxfoundation.org](http://www.linuxfoundation.org/)") as it was also present in the other blog posts.

**Comment by lpinca** [comment]
@nodejs/website can this be merged?

**Comment by thefourtheye** [comment]
cc @nodejs/ctc @nodejs/tsc

File: locale/en/blog/announcements/nodejs-foundation-momentum-release.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+category: Annoucements
+slug: node.js-foundation-momentum-release
+layout: blog-post.hbs
+---
+
+> With 5.7 million users; increased community participation and a solid Foundation backing: 2016 was a good year for the platform.
+
+**NODE.JS INTERACTIVE 2016, AUSTIN, TX., Nov. 30, 2016** — [The Node.js Foundation](https://nodejs.org/en/foundation/), a community-led and industry-backed consortium to advance the development of the Node.js platform, today announced [Snyk](https://snyk.io) as a Silver Member, major community and code growth, and the opening of an expansive Node.js user survey.
+
+Founded in 2015, the Node.js Foundation was created to accelerate the development of Node.js and support the large ecosystem that encompasses it through open governance. Membership has grown 30 percent growth since the Foundation’s inception and represents a mix of Fortune 500 companies and startups alike. The newest member Snyk is a security company that finds, fixes and monitors known vulnerabilities in Node.js and Ruby on Rails applications.
```
Comment (thefourtheye) [inline locale/en/blog/announcements/nodejs-foundation-momentum-release.md:13]: Can this be changed "has grown 30 percent growth"?

File: locale/en/blog/announcements/nodejs-foundation-momentum-release.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+About Node.js Foundation
+Node.js is used by tens of thousands of organizations in more than 200 countries and amasses more than 4.5 million active users per month. It is the runtime of choice for high-performance, low latency applications, powering everything from enterprise applications, robots, API engines, cloud stacks, and mobile websites.
+
+The Foundation is made up of a diverse group of companies including Platinum members GoDaddy, IBM, Intel, Joyent, Microsoft, NodeSource, PayPal, and Red Hat. Silver members include Apigee, AppDynamics, Cars.com, Codefresh, DigitalOcean, Dynatrace, Fidelity, Google, Groupon, nearForm, New Relic, npm, Opbeat, RisingStack, Sauce Labs, SAP, Snyk, StrongLoop (an IBM company), Sphinx, YLD, and Yahoo!. Get involved here: [https://nodejs.org](https://nodejs.org/).
+
+The Node.js Foundation is a Linux Foundation Project, which are independently funded software projects that harness the power of collaborative development to fuel innovation across industries and ecosystems. [www.linuxfoundation.org](http://www.linuxfoundation.org/)
```
Comment (thefourtheye) [inline locale/en/blog/announcements/nodejs-foundation-momentum-release.md:42]: Does this sentence look grammatically okay?

File: locale/en/blog/announcements/nodejs-foundation-momentum-release.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+During the conference, the Node.js Foundation also launched **its second user survey**, which will remain open until the end of December. This survey builds on the questions asked in the first Node.js Foundation survey, which was conducted in January 2016, and adds a number of questions designed to shed even more light on who uses Node.js, where they are located, how they learned Node.js, what they use it for, what other technologies they use it with and more.
+
+As with the January survey, the Node.js Foundation will produce and make available publicly for free a report with the survey’s key findings. If you use Node.js, please take the survey and tell your friends and colleagues: [https://www.surveymonkey.com/r/Node16](https://www.surveymonkey.com/r/Node16)
+
+If you are interested in receiving the latest updates from the conference and what’s to come in 2017, be sure to follow [@nodejs](https://twitter.com/nodejs) on Twitter and subscribe to the [Node.js Foundation mailing list](http://go.linuxfoundation.org/l/6342/2015-09-15/2sgqpp).
+
+About Node.js Foundation
```
Comment (thefourtheye) [inline locale/en/blog/announcements/nodejs-foundation-momentum-release.md:37]: Is this a heading?

File: locale/en/blog/announcements/nodejs-foundation-momentum-release.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+title: The Node.js Platform and Node.js Foundation Continue to Grow
+date: 2016-11-30T12:00:00.000Z
+category: Annoucements
+slug: node.js-foundation-momentum-release
+layout: blog-post.hbs
+---
+
+> With 5.7 million users; increased community participation and a solid Foundation backing: 2016 was a good year for the platform.
+
+**NODE.JS INTERACTIVE 2016, AUSTIN, TX., Nov. 30, 2016** — [The Node.js Foundation](https://nodejs.org/en/foundation/), a community-led and industry-backed consortium to advance the development of the Node.js platform, today announced [Snyk](https://snyk.io) as a Silver Member, major community and code growth, and the opening of an expansive Node.js user survey.
```
Comment (thefourtheye) [inline locale/en/blog/announcements/nodejs-foundation-momentum-release.md:11]: I would prefer breaking these lines at 80 characters.

File: locale/en/blog/announcements/nodejs-foundation-momentum-release.md (hunk 3 above)
Comment (lpinca) [reply locale/en/blog/announcements/nodejs-foundation-momentum-release.md:37]: This is consistent with other blog posts where a heading is not used here.

File: locale/en/blog/announcements/nodejs-foundation-momentum-release.md (hunk 2 above)
Comment (lpinca) [reply locale/en/blog/announcements/nodejs-foundation-momentum-release.md:42]: Copied from other blog posts.

File: locale/en/blog/announcements/nodejs-foundation-momentum-release.md (hunk 4 above)
Comment (lpinca) [reply locale/en/blog/announcements/nodejs-foundation-momentum-release.md:11]: While I like to have 80 chars per line, most of the markdown files in this repo don't follow this practice.

**Comment by ZibbyKeaton** [comment]
@thefourtheye @lpinca thank you!

**Review by hackygolucky — APPROVED** [review]
It looks like the requested changes were made minus a few word choices. Another press release for archiving!

**Comment by ZibbyKeaton** [comment]
@bnb mind helping us merge this and posting to the site?

**Pull request merged**